plugins/inara.py

```diff
@@ -361,6 +369,24 @@ def journal_entry(  # noqa: C901, CCR001
 
     :return: str - empty if no error, else error string.
     """
+    if not monitor.is_live_galaxy():
+        # This only applies after Update 14, which as of 2022-11-27 is scheduled
+        # for 2022-11-29, with the game servers presumably being down around
+        # 09:00
+        if datetime.now(timezone.utc) >= datetime.fromisoformat("2022-11-27T09:00:00+00:00"):
```

Comment:
I would move this into the other side, that way it's more generally useful

Reply:
I actually moved it *out* of that side so as to keep the semantics of `monitor.is_live_galaxy()` cleaner.

Long-term it'll actually get removed.  We just need it for this release due to making it before Update 14 and needing it to only have effect *after*.

This is where we're thankful that we only (normally) process live, new, Journal files.  Anyone playing silly buggers with moving files around gets to keep the broken pieces.

Reply:
Works for me